src/main/scala/nl.knaw.dans.easy.properties/app/repository/package.scala

```diff
@@ -31,7 +32,11 @@ package object repository {
   case class DepositDoesNotExistError(depositId: DepositId) extends QueryError(s"Deposit $depositId does not exist.")
   case class InvalidValueError(override val msg: String) extends QueryError(msg)
   object InvalidValueError {
-    def apply(ts: Seq[Throwable]): InvalidValueError = InvalidValueError(ts.map(_.getMessage).mkString("\n"))
+    def apply(ts: Seq[Throwable], query: String =""): InvalidValueError = {
+      val msg = ts.map(_.getMessage).mkString("\n")
+      logger.debug(s"$msg [$query]")
```

Comment:
if `query` is empty, you get `"<some msg> []"` here.

Reply:
Probably drop it for now and if you'd like, try to redesign the exceptions classes in another PR (rather than this one).